src/modules/collection/utils.ts

```diff
@@ -64,3 +64,7 @@ export function canMintCollectionItems(collection: Collection, address?: string)
 export function canManageCollectionItems(collection: Collection, address?: string) {
   return isOwner(collection, address) || isManager(collection, address)
 }
+
+export function hasReviews(collection: Collection) {
```

Comment:
I would add a message here explaining that this solution is not 100% accurate since for example metadata, or price, or beneficiary update can update this value too.